__tests__/server.test.js

```diff
@@ -287,3 +287,29 @@ describe("/api/articles/:article_id/comments", () => {
 		});
 	});
 });
+
+describe("/api/comments/:comment_id", () => {
+	describe("DELETE", () => {
+		test("DELETE 204, responds with a status of 204 and no content when a comment is deleted", () => {
+			return request(app).delete("/api/comments/4").expect(204);
+		});
+		test("DELETE 400, responds with a 400 error when requested with wrong data type", () => {
+			() => {
```

Comment:
I cant imagine this test runs, you've nested arrow functions here

Reply:
It wont fail because there isn't an expect that will fail as none ever got ran.

How did you ever get the `RED` in `RED, GREEN` here?

Reply:
oops, it does still run 🤷.. but point taken on the RED, GREEN.. I have been logging the error in app.js so not specifically RED, GREEN.. I'll start to remove the specific error codes from my error handler to achieve RED going forward 😊